src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -770,6 +771,15 @@ services:
   PrestaShopBundle\Form\Admin\AdvancedParameters\AuthorizationServer\ApplicationType:
   PrestaShopBundle\Form\Admin\Improve\International\Tax\TaxRulesGroupType:
 
+  PrestaShopBundle\Form\Admin\Sell\CartRule\:
```

Comment:
Oh this is nice! We could maybe do the same on some other namespaces to simplify the content of this file